CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ All notable changes to this project will be documented in this file.
 -   #1892 : Fix implementation of list function when an iterable is passed as parameter.
 -   #1924 : Fix internal error arising in Duplicate or list comprehensions.
 -   #1970 : Fix missing `TypeError` for wrong type passed as optional argument.
+-   #1297 : Fix iteration over an enum, map or zip in a functional for
```

Comment:
```suggestion
-   #1297 : Fix iteration over an enum, map or zip in a list comprehension.
```
Functional for is Pyccel terminology

Reply:
Done [03b6f4a](https://github.com/pyccel/pyccel/pull/1977/commits/03b6f4ae39a811bf68b7f5a66e9f9a6b4524171a)